cmd/xl-sets.go

```diff
@@ -1007,6 +1007,36 @@ func (s *xlSets) ReloadFormat(ctx context.Context, dryRun bool) (err error) {
 	return nil
 }
 
+// If it is a single node XL and all disks are root disks, it is most likely a dev setup, else it is a production setup.
+// On a dev setup we allow creation of format.json on root disks to help with testing.
+func isProductionSetup(infos []DiskInfo, errs []error) bool {
+	rootDiskCount := 0
+	for i := range errs {
+		if errs[i] != nil {
+			continue
```

Comment:
This can be a `break`?

Reply:
This doesn't need to be because for condition `rootDiskCount == len(errs)`